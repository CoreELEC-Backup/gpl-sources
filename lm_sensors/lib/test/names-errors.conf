# keyword with bogus names
label ?foo
label bar%
label baz$foo
label !
