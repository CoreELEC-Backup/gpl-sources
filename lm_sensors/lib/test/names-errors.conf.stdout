2: LABEL
2: ERROR
3: EOL
3: LABEL
3: NAME: bar
3: ERROR
4: EOL
4: LABEL
4: NAME: baz
4: ERROR
5: EOL
5: LABEL
5: ERROR
6: EOL
6: EOF
